src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,51 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+
+    Animal.alive.push(this);
+  }
 }
 
+Animal.alive = [];
+
 class Herbivore extends Animal {
-  // write your code here
+  constructor(
+    name,
+    health = 100,
+    hidden = false,
+  ) {
+    super(name, health);
+
+    this.hidden = hidden;
+
+    Animal.alive.push(this);
```

Comment:
Redundant!
This class extends Animal class. 
So, you push every Herbivore instance 2 times: in Animal constructor, and here.